doc/company.texi

```diff
@@ -1596,21 +1598,57 @@ system @uref{https://github.com/joaotavora/yasnippet, YASnippet}.
 @node Candidates Post-Processing
 @section Candidates Post-Processing
 
+@cindex post-processing
 @cindex sort
-@cindex duplicate
 @vindex company-transformers
-A list of completion candidates, supplied by a backend, can be
-additionally manipulated (reorganized, reduced, sorted, etc) before
-its output.  This is done by adding a processing function name to the
-user option @code{company-transformers} list, for example:
+A list of completion candidates supplied by backends can be
+manipulated before output: reorganized, reduced, sorted, etc.  To
+apply adjustments, add a processing function name to the user option
+@code{company-transformers} list.
+
+@cindex backends
+@cindex grouped backends
+@cindex :separate
+The transformer functions are called sequentially, each with the
```

Comment:
Using "in a sequence" instead of "sequentially" might make this a bit easier to read.